Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Projektarbeit-SSP\r\nwir bauen ein einfaches Schere Stein Papier Spiel in Python\r\n\r\n## Anforderungen\r\n\r\n\r\n\r\n- spielen gegen Computer\r\n- Namen des Spielers wählbar\r\n- es soll der Punktestand gespeichert werden\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision fbfd0ea29272d7891bb07bd4883914778f96b714)
+++ b/README.md	(date 1715173962138)
@@ -3,8 +3,6 @@
 
 ## Anforderungen
 
-
-
 - spielen gegen Computer
 - Namen des Spielers wählbar
-- es soll der Punktestand gespeichert werden
+- es soll der Punktestand gespeichert werden
\ No newline at end of file
